db/models/cart.js

```diff
@@ -0,0 +1,82 @@
+const client = require("./client");
+
+async function addItemToCart({ orderId, quantity }) {
+    try {
+        const { rows: [newOrder] } = await client.query(`
+        INSERT INTO cart("orderId", quantity)
```

Comment:
we can add orderId and quantity but the cart table definition has to reflect that.